lib/features/onboarding/presentation/page/onboarding_certificate_page.dart

```diff
@@ -28,27 +30,53 @@ class OnboardingCertificationPage extends ConsumerStatefulWidget {
 }
 
 class OnboardingCertificationPageState
-    extends BaseConsumerStatefulPageState<OnboardingCertificationPage> {
+    extends BaseConsumerStatefulPageState<OnboardingCertificationPage>
+    with ToastMixin {
   OnboardingCertificationPageState();
 
-  final TextEditingController _phoneController = TextEditingController();
+  final TextEditingController _codeController = TextEditingController();
```

Comment:
타입이 중복되어 노출되는 것 같아 아래와 같이 간단하게 작성해도 좋을 것 같습니다
```suggestion
  final _codeController = TextEditingController();
```